sunpy/sun/sun.py

```diff
@@ -74,7 +74,7 @@ def position(t='now'):
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
-def eccentricity_sunearth_orbit(t='now'):
+def eccentricity_sun_earth_orbit(t='now'):
```

Comment:
The next question becomes should we provide a deprecated wrapper for this?

Reply:
Fair question, I did add the breaking changelog just in case but maybe its not enough